src/components/PersonItem/PersonItem.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Link } from 'react-router-dom';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink/PersonLink';
+
+type Props = {
+  person: Person,
+  chosenUserSlug: string | undefined,
+};
+
+export const PersonItem: React.FC<Props> = ({ person, chosenUserSlug }) => {
+  const {
+    slug,
+    sex,
+    born,
+    died,
+    father,
+    mother,
+    fatherName,
+    motherName,
+  } = person;
+
+  return (
+    <tr
+      data-cy="person"
+      className={classNames({
+        'has-background-warning': slug === chosenUserSlug,
+      })}
+      key={slug}
+    >
+      <td>
+        <Link
+          to={`../${person.slug}`}
+          className={classNames({
+            'has-text-danger': person.sex === 'f',
+          })}
+        >
+          {person.name}
+        </Link>
+      </td>
+
+      <td>{sex}</td>
+      <td>{born}</td>
+      <td>{died}</td>
+      <td>
+        {mother
+          ? <PersonLink person={mother} />
+          : motherName || '-'}
```

Comment:
```suggestion
        {mother
          ? <PersonLink person={mother} />
          : motherName || '-'
        }
```

Also, magic value `-`